stan/math/opencl/add.hpp

```diff
@@ -4,6 +4,8 @@
 #include <stan/math/opencl/matrix_cl.hpp>
 #include <stan/math/opencl/kernels/add.hpp>
 #include <stan/math/opencl/err/check_matching_dims.hpp>
+#include <stan/math/prim/meta.hpp>
+
 #include <CL/cl.hpp>
```

Comment:
Not reviewing yet and not really related to this PR but can we remove these `<CL/cl.h> includes? I feel like it would be enough to just include it in the context file right?

Reply:
We could, it can def go out of some files but it's sort of a case of 'include what you use' in other files.

Reply:
Lets leave if for now. Will review the PR sometime later today.